features/FxA-54-push-on-status-change/README.md

```diff
@@ -63,9 +63,14 @@ in response to the following actions:
 * User deletes their account
 * User resets their password
 
+When the fxa-auth-server receives a `410 Gone`
```

Comment:
Agreed, zapping the entire device record is sub-optimal because it will cause the device to disappear from the devices list, stop syncing, etc.  I think we should just clear all the push-related items in the device record, and let the device restore them.  @philbooth thoughts?

Otherwise I'm r+ on the changes here.